sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/CatalystConf.scala

```diff
@@ -32,6 +32,8 @@ trait CatalystConf {
   def optimizerInSetConversionThreshold: Int
   def maxCaseBranchesForCodegen: Int
 
+  def runSQLonFile: Boolean
```

Comment:
we have never formalized this, but the java convention I believe is for acronyms larger than 4 letters, use Pascal case. For acronyms shorter than 4, upper is prefered.